critiquebrainz/frontend/external/bookbrainz_db/identifiers.py

```diff
@@ -0,0 +1,123 @@
+from typing import List
+from brainzutils import cache
+import sqlalchemy
+import critiquebrainz.frontend.external.bookbrainz_db as db
+from critiquebrainz.frontend.external.bookbrainz_db import DEFAULT_CACHE_EXPIRATION
+
+def fetch_identifiers(identifier_set_id: int) -> List:
+    """
+    Fetch identifiers from the database.
+    Args:
+        identifier_set_id (int): Identifier set ID.
+    Returns:
+        List of identifiers.
+    """
+    if not identifier_set_id:
+        return None
+    
+    key = cache.gen_key('identifier', identifier_set_id)
+    identifiers = cache.get(key)
+    if not identifiers:
+        with db.bb_engine.connect() as connection:
+            result = connection.execute(sqlalchemy.text("""
+                SELECT
+                    iden.type_id as type_id,
+                    idtype.label as label,
+                    iden.value as value
+                FROM bookbrainz.identifier_set__identifier idens
+                LEFT JOIN bookbrainz.identifier iden on idens.identifier_id = iden.id
+                LEFT JOIN bookbrainz.identifier_type idtype on iden.type_id = idtype.id
+                WHERE idens.set_id = :identifier_set_id;
+                """), ({'identifier_set_id': identifier_set_id}))
```

Comment:
double-check this syntax. I think the `()` is unnecessary, and probably has no effect